app/containers/SearchContainer/index.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * SearchContainer
+ *
+ */
+
+import React, { memo, useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage as T } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { selectSearchContainer, selectTrackData, selectTrackError, selectSearchTerm } from './selectors';
+import searchContainerSaga from './saga';
+import { Input } from 'antd';
+import get from 'lodash/get';
+import { isEmpty } from 'lodash';
+import debounce from 'lodash/debounce';
+import { searchContainerCreators } from './reducer';
+
+const { Search } = Input;
+
+export function SearchContainer({
+  dispatchGetTrackList,
+  dispatchClearTrackInfo,
+  intl,
+  searchTerm,
+  trackError,
+  trackData
```

Comment:
both trackData and trackError should be plural since these properties deal with an array of tracks